app/models/schema-block.ts

```diff
@@ -0,0 +1,18 @@
+import { attr } from '@ember-decorators/data';
+
+import OsfModel from './osf-model';
+
+export default class SchemaBlockModel extends OsfModel {
+    @attr('string') blockType!: string;
+    @attr('string') chunkId!: string;
+    @attr('string') answerId!: string;
+    @attr('string') displayText!: string;
+    @attr('string') helpText!: string;
+    @attr('boolean') required!: boolean;
```

Comment:
Per @jamescdavis, let's mark every attr `?` instead of `!` -- don't want to assert/assume a value where there might be none. It'll force us to handle the `undefined` case, which will be mildly annoying but will make our code more robust!